metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -239,4 +241,18 @@ entities:
     keyAspect: inviteTokenKey
     aspects:
       - inviteToken
+  - name: corpRole
+    doc: CorpRole represents a role define in a database.
+    keyAspect: corpRoleKey
+    aspects:
+      - corpRoleInfo
```

Comment:
We need some aspect to define the origin of the role - this has bitten us in the past. That is - If the role is external (coming from another system) we need some way to define that